core/pom.xml

```diff
@@ -38,6 +38,21 @@
         <scope>test</scope>
     </dependency>
 
+      <dependency>
+        <groupId>com.vlkan.hrrs</groupId>
+        <artifactId>hrrs-servlet-filter-base64</artifactId>
+        <version>0.5</version>
+      </dependency> 
+      <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>
+        <version>23.6-jre</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
+        <artifactId>httpcore</artifactId>
+        <version>4.4.8</version>
```

Comment:
we are currently importing httpcore 4.3.2 in the root pom.xml .. perhaps we should upgrade that rather than load two separate versions of httpcore.